KEYS

```diff
@@ -960,3 +960,34 @@ uVrQ13BH7D1TWlzM7jN1dToIySXIjGRcMpqT/2LqYDwHfrJCdQZGKDBl5vNWXwNs
 OC7PFtJwtXNl8p7Jvk3lplU5rNSSkTF8hSFgtxu4cpJ5VAY8nQ==
 =FdGa
 -----END PGP PUBLIC KEY BLOCK-----
+
+pub   ed25519 2022-10-19 [SC] [expires: 2024-10-19]
```

Comment:
Why does the key expire? Will this mean that people can't verify ICU 72 binaries in two years?

Reply:
It'll always remain possible to verify the signature, it'll then just say that the key has since expired.